modules/data.remote/tests/testthat/test.socextract.R

```diff
@@ -0,0 +1,10 @@
+context("*** Test soilgrids.soc.extract function at one location ***")
+
+test_that("The soilgrids.soc.extract function properly obtains data for one lat/long", {
+  site_info <- data.frame(id=676, sitename="Willow Creek (US-WCr)", lat=45.805925, lon=-90.07961,
```

Comment:
1. this would be a good example (see previous comment about dropping bety query
2. can we explicitly set the API-based tests in this PR to not run with every PR build? Doing so has a tendency to break the CI

Reply:
added "skip_on_ci()" in the tests that use the API